docs/developing.rst

```diff
@@ -151,23 +151,22 @@ ParameterHandler
 
 .. TODO : fill in the modular components below
 
-ParameterType
+    ParameterType
 
-   ToolkitRegistry
-
-       ``ToolkitRegistry.from_object``  / ``ToolkitRegistry.from_smiles`` / ``OpenEyeToolkitWrapper.from_openeye`` / ``RDKitToolkitWrapper.from_rdkit``
+    ToolkitRegistry
+        ``ToolkitRegistry.from_object``  / ``ToolkitRegistry.from_smiles`` / ``OpenEyeToolkitWrapper.from_openeye`` / ``RDKitToolkitWrapper.from_rdkit``
         - These methods are a bit strange because they are effectively classmethods for ``FrozenMolecule`` and ``Molecule`` subclasses.
           In `PR #583 <https://github.com/openforcefield/openff-toolkit/pull/583>`_, jaimergp raised a concern that effectively boils down to "if I subclass ``Molecule`` into a new class, ``MyMol``, then I expect ``MyMol.from_rdkit`` to return an instance of ``MyMol``, not ``Molecule``.
```

Comment:
I know this is ugly/comment-ish, but we should keep it visible in the compiled docs (since it's a source of very unexpected behavior due to our architecture choices)

Reply:
Done in 87c04bc